opentracing-api/src/main/java/io/opentracing/tag/Tags.java

```diff
@@ -45,17 +45,6 @@ private Tags() {
      */
     public static final String SPAN_KIND_CONSUMER = "consumer";
 
-    /**
-     * The service name for a span, which overrides any default "service name" property defined
-     * in a tracer's config. This tag is meant to only be used when a tracer is reporting spans
-     * on behalf of another service (for example, a service mesh reporting on behalf of the services
-     * it is proxying). This tag does not need to be used when reporting spans for the service the
-     * tracer is running in.
-     *
-     * @see #PEER_SERVICE
-     */
-    public static final StringTag SERVICE = new StringTag("service");
```

Comment:
Why can't this be deprecated instead of removed?

Reply:
+1 here. This would be breaking things.

Reply:
This tag was not present in `0.31.0`, so this would only be a breaking change for someone using a unofficial build.  That is the main reason I'm pushing to get it removed before `0.32.0` so it doesn't have to go through a deprecation period.